### ISSUE TITLE
These PU issue cross-links should be removed!!!

### ISSUE THREAD

**Issue by mattbeasley96** [dotnet/AspNetCore.Docs#27921]
We spent 3+ hours looking and trying to follow the reference articles in the MICROSOFT documentation: https://learn.microsoft.com/en-us/aspnet/core/security/authentication/scaffold-identity?view=aspnetcore-7.0&tabs=visual-studio#custom-identity-components

The articles that are there as examples are outdated and DO NOT work with the latest builds.  

[Support Custom Login Component when using Identity (dotnet/aspnetcore #13601)](https://github.com/dotnet/aspnetcore/issues/13601) 

This article and example is for ASP.NET Core 3.1x and has no significant value to anyone working with .NET 6 or 7.  
These two also only will work on ASP.NET 3.1x:
[Reiteration on the SigninManager<T> not being supported in Razor Components (dotnet/aspnetcore #34095)](https://github.com/dotnet/aspnetcore/issues/34095)
[There is no info on how to actually implement custom login form for server-side blazor (dotnet/AspNetCore.Docs #16813)](https://github.com/dotnet/AspNetCore.Docs/issues/16813)

Again, a complete waste of time to review.   My fellow developers and I are baffled on how Microsoft can still have documentation on ASP.NET Core & Blazor site that does not work or that the issues have been closed in Github for a year or more.  Please remove these links to save other developers hours of wasted time.

Best regards,
Frank Galarraga (@fgalarraga) & Matt Beasley (@mattbeasley96)

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 804e24dd-7e30-8cf5-2c99-1ffacb4582f6
* Version Independent ID: 3e25547b-e2c9-6634-2c38-2a77207f17d4
* Content: [Scaffold Identity in ASP.NET Core projects](https://learn.microsoft.com/en-us/aspnet/core/security/authentication/scaffold-identity?view=aspnetcore-7.0&tabs=visual-studio#scaffold-identity-into-a-blazor-server-project)
* Content Source: [aspnetcore/security/authentication/scaffold-identity.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authentication/scaffold-identity.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
Hello @mattbeasley96 ... I'll address your concerns in the morning. It's late here ... and Friday to boot. I'll get back to you.

**Comment by guardrex** [comment]
btw tho ... That's a bit hard to follow. When you wrote ...

> This article and example

... can you be specific? What is linked under the text ...

> For additional context and further assistance, explore the following discussions and resources:

... are PU issues/discussions, not articles. We don't support developer code in such discussions. If the PU suggested something that you're referring to, then please cross-link what you're describing.

In fact, there are ***NO*** articles or examples that Microsoft has put out that describe the concept of custom Identity Razor components. The PU decided not to pursue it. It isn't supported. There are third-party components for it. There are blogs in the Net for it. We don't support such documentation.

WRT what's linked ... Yes, I will review them. The idea behind linking them is that ... as you can see ... devs wrote in a few times asking about this, discussing it with the product unit, and perhaps not getting the answer that they wanted. They mostly want MS to build them into the framework. The PU has refused to do so. The discussions surface that fact, so that's why I cross-linked them.

Still tho ... as for anything that you saw saying 'do it like this' with some code ... those aren't Microsoft folks. Any such code is just from the community and not supported by MS.

**Comment by Rick-Anderson** [comment]
>https://github.com/dotnet/aspnetcore/issues/13601

> This article and example is for ASP.NET Core 3.1x and has no significant value to anyone working with .NET 6 or 7.

I disagree, ASP.NET Core 3.1 hasn't changed that much in 6 & 7. The link is for advanced programmers as a hint. Issues are not kept up to date.

**Comment by Rick-Anderson** [comment]
@guardrex  wrote:

> WRT what's linked ... Yes, I will review them. T

Perhaps just add, the following links  are dated and not supported but might be helpful to some developers.

**Comment by guardrex** [comment]
I agree with that, @Rick-Anderson.

I will do something with this to make it clear that these are the discussions where the PU decided not to pursue it.

Or ... indeed ... as @mattbeasley96 suggests ... we don't really need to cross-link issues where the PU doesn't say anything helpful at all and just tells everyone to take a hike! 😆 lol ... jk jk ... Javier has been firm on this point for many years now. There will be no Identity Razor components. I don't expect that to change, nor have they provided any particularly helpful advice on creating them. The topic section here tells devs what to do tho ... use web API to a backend endpoint to process things for "Identity" (in quotes) Razor components. That's what the PU wants folks to do, and that's what this says.

I'll look at this further on Monday morning.
